prs/WeatherData.test.cs

```diff
@@ -0,0 +1,9 @@
+namespace Sample {
+    [TestClass]
+    public class WeatherDataTest {
+        [TestMethod]
+        public static void Test1 {
+            // TODO: write tests 😝
```

Comment:
Should do the unit-test on updateMeasurements() and updateTarget() within the same PR.